modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/TxManagerImpl.java

```diff
@@ -525,12 +525,12 @@ private static long physicalExpirationTimeMillis(HybridTimestamp beginTimestamp,
     }
 
     private static long sumWithSaturation(long a, long b) {
-        assert a >= 0 : a;
-        assert b >= 0 : b;
+        assert a >= USE_CONFIGURED_TIMEOUT_DEFAULT : a;
+        assert b >= USE_CONFIGURED_TIMEOUT_DEFAULT : b;
 
         long sum = a + b;
 
-        if (sum < 0) {
+        if (sum < USE_CONFIGURED_TIMEOUT_DEFAULT) {
```

Comment:
In this code, 0 means just zero (it's an arithmetic operation), so this change should probably be reverted

Reply:
Woop, the IDEA did something wrong, rolled it back.